PEMPal.1.0/Views/HomePageView.swift

```diff
@@ -80,6 +81,131 @@ struct HomePageView_Previews: PreviewProvider {
 //    }
 //}
 
+struct PEMStatus: View {
+    // might need to re-scope Health Kit code to HomePageView struct instead of here in order to enable access for other vital struct boxes in PEM Info
+    
+    private var healthStore = HKHealthStore()
+    let heartRateQuantity = HKUnit(from: "count/min")
+    
+    
+    let store = HKHealthStore()
+        
+    var updateView: (() -> Void)?
+    
+    @StateObject var globalModel = GlobalModel()
+    
+    // boolean that evaluates whether or not user is at risk for PEM based on heart rate measurements
+    // currently buggy - globalModel.userAge always seems to return nil, so the program defaults to 120 bpm as the threshold rate
+    var riskBoolean: Bool {
+        if globalModel.userAge != nil {
+            let age = globalModel.userAge
+            if globalModel.userGender == "F" {
+                return Double(globalModel.hRValue) > Double(220 - (Double(age!) * 0.88)) * 0.6
+            } else {
+                return Double(globalModel.hRValue) > Double(220 - age!) * 0.6 // use lower estimate if gender is not female
+            }
+        } else {
+            return globalModel.hRValue > 120 // arbitrary threshold if no age or gender is supplied
+        }
+    }
+    
+    var riskColor: Color {
+        return riskBoolean ? Color.red : Color.green
+    }
+    
+    var buttonLabel: String {
+        return riskBoolean ? "High Risk 🚨" : "Low Risk ✅ "
+    }
+    
+    var body: some View{
+        Button{
+        } label: {
+            Text (buttonLabel)
+                .frame(width: 300, height: 100, alignment: .center)
+                .font(.system(size: 42, weight: .semibold, design: .rounded))
+                //.fontWeight(.semibold)
+                .padding(.horizontal, 10.0)
+                .padding(.vertical, 2.0)
+                .foregroundColor(.white)
+                .background(riskColor)
+                .cornerRadius(8)
+        }
+        .onAppear(perform: start)
+    }
+    func start() {
```

Comment:
Everything here and below we don't want to do because globalModel is already querying